gerardo-santos/react-apprenticeship-capstone-2
# Feat fetch on change

## Description
```
In this pull request the picture of the day was fetched on change.

- Previously the picture of the day was fetched on a button click, now on change.
```
## Type of Change

```
Delete options that are not relevant.
```



## How should this be tested?
```
Verifying the data is fetched on change.

- Check the picture is fetched correctly.
- Check that error messages are shown when an error occurs.
```
## Any background context you want to provide?
```
N/A
```
## What is the deliverable?
```
Picture of the day fetched using debounce.
```
## Screenshots and links
```
https://nostalgic-lichterman-b0664b.netlify.app/
```
## Notes
```
N/A
```
## Checklist
N/A
